### DISCUSSION TITLE
Cursor style is resetting

### DISCUSSION THREAD

**Discussion by nikey02** [termux/termux-app#2283] [General]
I have changed cursor style to bar in `termux.properties`
But when i open `nvim` and close it 
It get back to `block`

Normal :
![IMG_20210920_180324](https://user-images.githubusercontent.com/68458326/134003338-db0fe114-032d-4a70-85c3-828a2039173d.jpg)

After using nvim:
![IMG_20210920_180417](https://user-images.githubusercontent.com/68458326/134003408-d4b5d822-6fed-404b-848b-a5d42ea42287.jpg)

**Comment by agnostic-apollo** [comment]
It's likely because `nvim` sets its own cursor style and it can't restore original automatically on exit and neither can termux.

Check https://github.com/neovim/neovim/wiki/FAQ#how-to-change-cursor-shape-in-the-terminal and https://vi.stackexchange.com/questions/25103/neovim-does-not-restore-terminal-cursor-on-exit to solve your issue.

**Comment by RalfWerner** [comment]
would `termux-reload-settings` not restore the _termux style?_

**Reply by agnostic-apollo** [reply]
No, but you can use:

- block: `echo -e "\033[2 q"`
- underline: `echo -e "\033[4 q"`
- bar: ` echo -e "\033[6 q"`
